### PR TITLE
Add supplementary care standard

### PR DESCRIPTION
**Pull request by ashleywntrbjss**
This PR:

- Adds supplementary care standard type to standardtypes enum and standardtypes table
- Adds supplementary care standard expander to standards page on catalogue solution
- Updates the existing standard expanders to use a common partial component
- Updates the function app importer to map to the new standard type
- Removes the migration file as all standards appear to be migrated to the new standard types
- Adds test data:
    - Single supplementary care standard as found on the confluence
    - Work off plans for a few solutions to allow in progress indicator to appear against standard
    - Modified set of csv files for function app import available in dev ops ticket

### CODE REVIEW

File: database/NHSD.GPITBuyingCatalogue.Database/PostDeployment/TestData/InsertWorkOffPlans.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- IF UPPER('$(INSERT_TEST_DATA)') = 'TRUE'
```
Comment (ghost) [inline database/NHSD.GPITBuyingCatalogue.Database/PostDeployment/TestData/InsertWorkOffPlans.sql]: This condition will need to be uncommented again so it doesn't add records in live

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/StandardType.cs [hunk 2]
```diff
@@ -9,6 +9,10 @@ public enum StandardType
         [Display(Name = "Overarching")]
         Overarching = 1,
 
+        [Description("SupplementaryCare")]
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/StandardType.cs]: It should be fine to space these words apart. We've had similar instances in the past, such as the `Context Specific` standard type that was recently removed.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml [hunk 3]
```diff
@@ -21,84 +42,13 @@
...
-                else
-                {
-                    <nhs-table-cell><nhs-tag colour="Green" text="Fully&nbsp;met" /></nhs-table-cell>
-                    <nhs-table-cell></nhs-table-cell>
-                }
-            </nhs-table-row-container>
-        }
-    </nhs-table>
-</nhs-expander>
+<partial name="_StandardsExpander" model="overarchingStandardsExpanderModel"/>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml:45]: Very nice - this cleans the page up a lot

**Review by bjss-colin — CHANGES_REQUESTED** [review]
Several minor points. I'll pull these changes down and do some testing now

File: database/NHSD.GPITBuyingCatalogue.Database/PostDeployment/TestData/InsertWorkOffPlans.sql (hunk 1 above)
Comment (ashleywntrbjss) [reply database/NHSD.GPITBuyingCatalogue.Database/PostDeployment/TestData/InsertWorkOffPlans.sql]: Thanks for spotting this! Fixed

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**